src/plugins/notebook/components/notebook-embed.vue

```diff
@@ -165,19 +165,32 @@ export default {
             }).show(this.embed.snapshot.src);
         },
         changeLocation() {
-            this.openmct.time.stopClock();
-            this.openmct.time.bounds({
-                start: this.embed.bounds.start,
-                end: this.embed.bounds.end
-            });
-
             const link = this.embed.historicLink;
             if (!link) {
                 return;
             }
 
+            const bounds = this.openmct.time.bounds();
+            const isTimeBoundChanged = this.embed.bounds.start !== bounds.start
+                && this.embed.bounds.end !== bounds.end;
+            const isFixedTimespanMode = !this.openmct.time.clock();
+
+            this.openmct.time.stopClock();
```

Comment:
why do we need to stopClock?

Reply:
Ahh, was under impression that stop puts time conductor in fixed mode and then change bounds. But looks like directly change bounds also works (line 183). Let me remove, stopClock